Android/Guava/source/Guava/Guava.csproj

```diff
@@ -28,18 +30,33 @@ Guava is a set of core libraries that includes new collection types (such as mul
   </PropertyGroup>
 
   <ItemGroup>
-    <None Include="Guava.targets" Pack="True" PackagePath="@(_TfmNuGetBuildFolders->'%(Identity)$(PackageId).targets')" />
+    <None Remove="Xamarin.Google.Code.FindBugs.JSR305" />
+    <None Remove="Xamarin.Google.ErrorProne.Annotations" />
+    <None Remove="Xamarin.CodeHaus.Mojo.AnimalSnifferAnnotations" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="Guava.targets" Pack="True" PackagePath="@(_TfmNuGetBuildFolders-&gt;'%(Identity)$(PackageId).targets')" />
     <None Include="..\..\externals\guava.jar" Pack="True" PackagePath="jar" />
     <None Include="..\..\External-Dependency-Info.txt" Pack="True" PackagePath="THIRD-PARTY-NOTICES.txt" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\Guava.FailureAccess\Guava.FailureAccess.csproj" PrivateAssets="None" />
     <ProjectReference Include="..\Guava.ListenableFuture\Guava.ListenableFuture.csproj" PrivateAssets="None" />
+    <ProjectReference Include="..\Xamarin.CheckerFramework.CheckerCompatQual\Xamarin.CheckerFramework.CheckerCompatQual.csproj" />
+    <ProjectReference Include="..\Xamarin.CodeHaus.Mojo.AnimalSnifferAnnotations\Xamarin.CodeHaus.Mojo.AnimalSnifferAnnotations.csproj" />
```

Comment:
We don't need these `<ProjectReference>`'s or the `.csproj`'s for the dependencies we are pulling from NuGet instead of creating bindings for.

Reply:
Fixed.

IIRC - I removed them in VS4Mac (see  `Remove None=`, but `ProjectReference`s were not removed and I did not check `csproj` after that.

This might be VS4Mac bug. Must try again, but when this is over.